tests/desktop/test_paypal.py

```diff
@@ -18,12 +18,10 @@ class TestPaypal:
     addon_name = 'Firebug'
 
     @pytest.mark.login
-    def test_that_user_can_contribute_to_an_addon(self, mozwebqa):
+    def test_that_user_can_contribute_to_an_addon(self, mozwebqa, existing_user, paypal):
```

Comment:
Perhaps `paypal_user` would be a bit more descriptive of a variable name than just `paypal`.